docs/_docs/04_reference/05-strict-mode.md

```diff
@@ -53,84 +53,116 @@ $ TG_STRICT_MODE='true' terragrunt plan-all
 ```
 
 Instead of enabling strict mode like this, you can also enable specific strict controls by setting the [strict-control](/docs/reference/cli-options/#strict-control)
-flag to a value that's specific to a particular strict control.
+flag to a value to a particular strict control.
+
 This can allow you to gradually increase your confidence in the future compatibility of your Terragrunt usage.
 
 ```bash
-$ terragrunt plan-all --strict-control deprecated-commands
+$ terragrunt plan-all
 15:26:08.585 WARN   The `plan-all` command is deprecated and will be removed in a future version. Use `terragrunt run-all plan` instead.
 ```
 
 ```bash
-$ terragrunt plan-all --strict-control deprecated-commands
+$ terragrunt plan-all --strict-control plan-all
```

Comment:
What is the point of the user uses this:

```
terragrunt plan-all --strict-control plan-all
```

to enable strict control for the `plan-all` command, and then to intentionally specify the `plan-all` command itself. At the same time, knowing that he will 100% get an error.
In my understanding, strict control should help the user from unintentionally using deprecated commands and flags.

That's why it makes more sense to use this command

```
terragrunt plan-all --strict-control legacy-all
```